Marlin/src/module/planner.h

```diff
@@ -254,10 +265,14 @@ typedef struct PlannerBlock {
 
   // Advance extrusion
   #if ENABLED(LIN_ADVANCE)
-    uint32_t la_advance_rate;               // The rate at which steps are added whilst accelerating
-    uint8_t  la_scaling;                    // Scale ISR frequency down and step frequency up by 2 ^ la_scaling
-    uint16_t max_adv_steps,                 // Max advance steps to get cruising speed pressure
-             final_adv_steps;               // Advance steps for exit speed pressure
+    #if ENABLED(SMOOTH_LIN_ADV)
+      bool use_advance_lead;
+    #else
+      uint16_t max_adv_steps,                 // Max advance steps to get cruising speed pressure
+              final_adv_steps;               // Advance steps for exit speed pressure
+      uint8_t  la_scaling;                    // Scale ISR frequency down and step frequency up by 2 ^ la_scaling
+      uint32_t la_advance_rate;               // The rate at which steps are added whilst accelerating
```

Comment:
This would be tidier if the code was an exact copy of the original except for the extra indentation. Would also fix the misalignment in the comments.

Reply:
ups...